src/main/java/org/dita/dost/module/GenMapAndTopicListModule.java

```diff
@@ -901,6 +906,27 @@ private FileInfo getOrCreateFileInfo(final Map<URI, FileInfo> fileInfos, final U
         fileInfos.put(i.src, i);
         return i;
     }
+    
+    /** Treat flag image references as if they are relative to the input dir.
+     * @param fileInfos
+     * @param file
+     * @param file
+     * @return
+     */
+    private FileInfo getOrCreateFlagImageFileInfo(final Map<URI, FileInfo> fileInfos, final URI file, final URI relpath) {
+        assert file.getFragment() == null;
+        final URI f = file.normalize();
+        FileInfo.Builder b;
```

Comment:
`b` should be `final`

Reply:
This method was based on the more general `getOrCreateFileInfo` one right above it -- is there a reason that one doesn't use `final`, or should that be changed too? (When I change either of them to `final`, I get compiler warnings a few lines down setting `b.uri()`). Lines 893 and 899 in the latest: https://github.com/dita-ot/dita-ot/blob/develop/src/main/java/org/dita/dost/module/GenMapAndTopicListModule.java#L893

Reply:
(Wondering if the `.uri()` should be moved up for each of these methods - seems they should be consistent)